Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.0'
+gem 'rails', '~> 4.2.7.1'
```

Comment:
I looked like brakeman was complaining, but I re-checked and 4.2.8 also seems to work. Will update.